src/hooks/useAccountBalance.ts

```diff
@@ -88,6 +98,9 @@ export const useAccountBalance = ({
     },
   });
 
+  // const solanaNative = useSolanaNativeBalance(solAddress) // May not need this, depends on bridge flow
```

Comment:
[q] can we expand on this comment? i'm not sure what this means

Reply:
Deleting this comment!